enterprise/server/cmd/ci_runner/main.go

```diff
@@ -993,6 +1004,70 @@ func (ar *actionRunner) Run(ctx context.Context, ws *workspace) error {
 		}
 	}
 
+	// TODO(Maggie): Emit BES events for each bazel command
+	for i, step := range action.Steps {
+		// TODO(Maggie): Provision one sub-dir per bazel command
```

Comment:
I think the current approach here seems OK? (one subdir per bash command)

Reply:
I wasn't sure if the artifacts had to be in separate dirs for how we show them in the UI, so added a TODO so I didn't forget to look into it. Anyway just removed it

Reply:
Ah right, we render them as `bazel_commands[0]`. Maybe we could change these to `steps[0]` etc. once we are migrated